_pages/en/it-picture-medium-term.md

```diff
@@ -28,7 +28,27 @@ In the following sections, we're trying to take the same end state and create "v
 
 #### Automatic Enrollment and Delivery
 
-- Number of services that require users to submit, or print, PDFs should be reduced by half
+Looking forward, the information required to enroll a Canadian into programs often exists via trusted organizations or other government departments, allowing for the benefits the programs to be made available without the need for a manual process on the behalf of citizens (more details can be found in the [long term plan](https://sara-sabr.github.io/ITStrategy/it-picture-long-term.html)). In order to move in this direction, IITB must move away from manual processes and towards automation and information systems allowing for the secure exchange of information. This aligns well with both the OneGC initiative and the [Policy on Service](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27916) which reads
+
+> 6.1 Deputy heads are responsible for applying the following principles to achieve better and more efficient design and delivery of Government of Canada services:
```

Comment:
citizen view does not need link to TBS Policy. Move  to IITB-ESDC section

Reply:
Hm ... Citizen view doesnt mean that a citizen will be reading this, it just means the view from a citizens perspective so ... 

It's our mgmt reading this, to understand what it could look like from a citizens perspective. Why we are proposing this as what it should look like from the citizens perspective is still relevant, no? Such as the relevant policies we're aligning with ...

Reply:
@jaysonmc see comment below on "GENERAL COMMENT ON STRUCTURE"
perhaps it's talking about the citizen's perspective first, then all the enablers. There will be many enablers to achieve info sharing and how it links to TB policies. I worry we will lengthen the document too much, repeating content between sections